src/EA.Weee.Web/Views/NewUser/TermsAndConditions.cshtml

```diff
@@ -25,14 +25,16 @@
         <li>"content" means all information and other materials on our service</li>
         <li>"document" means any reproducible artefact, file or printed copy of a document which is made as a result of the use of and information contained within our service which has been contributed by you</li>
         <li>
-            "online service" and "our service" means WEEE online service and is limited to the use of the on-line functionality provided by us to you to as an:
+            "online service" and "our service" means WEEE online service and is limited to the use of the on-line functionality provided by us to you as an:
             <ul>
                 <li>operator of a PCS to create an account as a user, join an organisation manage your organisation’s users, submit applications to register your Members and submit your quarterly returns of data to the appropriate authority</li>
                 <li>operator of an AATF or AE to create an account as a user, join an organisation, manage your organisation’s  users and submit your quarterly returns of data to the appropriate authority</li>
-                <li>the statutory duties and obligations required by the WEEE Regulations that apply to you or any appropriate authority are not affected by the use of this online service</li>
             </ul>
         </li>
     </ul>
+    <div>
+        <p style="margin-left: 5%;">The statutory duties and obligations required by the WEEE Regulations that apply to you or any appropriate authority are not affected by the use of this online service</p>
```

Comment:
Can you check if a special char is on the end of this P tag, some reason showing odd in the PR

Reply:
Had a look in the code and can't see anything out of the ordinary, also looks fine running it on my localhost